Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -470,16 +406,6 @@ public boolean onMenuItemClick(MenuItem item) {
         getListView().setMultiChoiceModeListener(this);
     }
 
-    private void updateSortOrder(String label, int index) {
-        AnalyticsTracker.track(
-            AnalyticsTracker.Stat.LIST_SORTBAR_MODE_CHANGED,
```

Comment:
Minor (🔍): Consider removing this unused tracking `Stat` as well: `LIST_SORTBAR_MODE_CHANGED`

Reply:
Good catch! Removed in 8f10c597c69df02abe3f343f433a70bbd47e9500.

Reply:
Perfecto! 🥇